frame/vamm/src/lib.rs

```diff
@@ -259,6 +264,18 @@ pub mod pallet {
 		/// is flaged to be closed and the closing action will take (or took)
 		/// effect at the time `timestamp`.
 		pub closed: Option<Moment>,
+
+		// TODO(Cardosaum): Write description for this field.
+		pub base_asset_twap: Balance,
+
+		// TODO(Cardosaum): Write description for this field.
+		pub base_asset_twap_timestamp: Moment,
+
+		// TODO(Cardosaum): Write description for this field.
+		pub quote_asset_twap: Balance,
+
+		// TODO(Cardosaum): Write description for this field.
+		pub quote_asset_twap_timestamp: Moment,
```

Comment:
Are these updated anywhere?

Reply:
Yeah they will, I just opted to open a new PR for it. :)